src/admin/views.py

```diff
@@ -0,0 +1,691 @@
+#
+# Plantalytics
+#     Copyright (c) 2016 Sapphire Becker, Katy Brimm, Scott Ewing,
+#       Matt Fraser, Kelly Ledford, Michael Limb, Steven Ngo, Eric Turley
+#     This project is licensed under the MIT License.
+#     Please see the file LICENSE in this distribution for license terms.
+# Contact: plantalytics.capstone@gmail.com
+#
+
+import json
+import logging
+import re
+import datetime
+import time
+
+from common.exceptions import *
+from common.errors import *
+from django.views.decorators.csrf import csrf_exempt
+from django.http import (
+    HttpResponse,
+    HttpResponseForbidden,
+    HttpResponseBadRequest,
+    HttpResponseServerError,
+    HttpResponseNotAllowed
+)
+
+import cassy
+
+logger = logging.getLogger('plantalytics_backend.admin')
+
+
+def verify_admin(admin_username, auth_token):
+    """
+    Verifies username has admin privileges and is authenticated.
+    """
+
+    try:
+        message = (
+            'Validating {} is an authenticated admin.'
+        ).format(admin_username)
+        logger.info(message)
+        is_admin = cassy.verify_authenticated_admin(admin_username, auth_token)
+        if (is_admin is False):
+            return False
+        message = (
+            'Successfully validated {} is an authenticated admin.'
+        ).format(admin_username)
+        logger.info(message)
+        return True
+    except Exception as e:
+        raise e
+
+
+@csrf_exempt
+def user_info(request):
+    """
+    Endpoint returns the user info for the requested username.
+    """
+
+    if request.method != 'POST':
+        return HttpResponseNotAllowed(['POST'])
+
+    data = json.loads(request.body.decode('utf-8'))
+
+    auth_token = str(data.get('auth_token', ''))
+    admin_username = str(data.get('admin_username', ''))
+    request_username = str(data.get('request_username', ''))
+
+    try:
+        is_verified = verify_admin(admin_username, auth_token)
+        if (is_verified is False):
```

Comment:
Could simplify these two lines
if not verify_admint(admin_username, auth_token): 

OR 

if not is_verified: